<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfig</class>
 <widget class="QWidget" name="DialogConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>331</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <widget class="QComboBox" name="comboBoxInterfacePort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>InterfacePort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLightPort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>LightPort</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Port Setting...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="title">
    <string>Camera Setting...</string>
   </property>
   <widget class="QLineEdit" name="lineEditResY">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>123</x>
      <y>129</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ResolutionY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>105</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ResolutionX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditResX">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>106</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vision Z
Offset</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVisionZ">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>-10.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>165</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Illuminaton</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sliderIllumination">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider::groove:horizontal</string>
    </property>
    <property name="minimum">
     <number>-13</number>
    </property>
    <property name="maximum">
     <number>-1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel #1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCam1Port">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>40</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCam2Port">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>70</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel #2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="sliderExposure">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal</string>
   </property>
   <property name="minimum">
    <number>-13</number>
   </property>
   <property name="maximum">
    <number>-1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClose">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>exposure</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAlign1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>142</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1Point</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAlign2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>142</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2Point</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>116</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Align Mode</string>
   </property>
   <widget class="QRadioButton" name="radioButtonAlign3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>26</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AllPoint</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>comboBoxInterfacePort</zorder>
  <zorder>label</zorder>
  <zorder>comboBoxLightPort</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButtonSave</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_3</zorder>
  <zorder>comboBoxCam1Port</zorder>
  <zorder>comboBoxCam2Port</zorder>
  <zorder>label_4</zorder>
  <zorder>sliderExposure</zorder>
  <zorder>pushButtonClose</zorder>
  <zorder>label_5</zorder>
  <zorder>radioButtonAlign1</zorder>
  <zorder>radioButtonAlign2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
